projects/experimental/components/badge/badge.template.html

```diff
@@ -0,0 +1,16 @@
+<div class="t-image">
+    <ng-content select="img"></ng-content>
+</div>
+<div
+    class="t-icon"
+    [class._icon-only]="!text.innerHTML"
```

Comment:
`.trim()`. Also I hope it does not cause any change detection issues?

Reply:
no issues, I checked with angular profiler